model/bindings.go

```diff
@@ -0,0 +1,122 @@
+package model
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	yaml "github.com/cloudfoundry-incubator/candiedyaml"
+	"github.com/docker/libcompose/config"
+	"github.com/docker/libcompose/utils"
+	"github.com/rancher/rancher-compose/preprocess"
+	"github.com/rancher/rancher-compose/rancher"
+	"io/ioutil"
+	"os"
+	"reflect"
+)
+
+type mapLabel map[string]string
+
+//BindingProperty holds bindings
+type BindingProperty struct {
+	Binding
+}
+
```

Comment:
Do we need the two level nested struct? BindingProperty.Binding.Scale? Can we just have Binding.Scale and return a map[string]Binding